jobs/config.json

```diff
@@ -6818,39 +6818,39 @@
       "UNKNOWN"
     ]
   },
-  "ci-kubernetes-e2e-gke-new-master-cvm-kubectl-skew": {
+  "ci-kubernetes-e2e-gke-new-master-gci-kubectl-skew": {
     "args": [
       "--check-leaked-resources",
       "--check-version-skew=false",
       "--env-file=jobs/platform/gke.env",
-      "--env-file=jobs/ci-kubernetes-e2e-gke-new-master-cvm-kubectl-skew.env",
+      "--env-file=jobs/ci-kubernetes-e2e-gke-new-master-gci-kubectl-skew.env",
```

Comment:
Once we merge https://github.com/kubernetes/test-infra/pull/3819 we can get rid of those env files entirely.